src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -274,7 +272,13 @@ file that was distributed with this source code.
                 return true;
             });
 
-            {% if sonata_admin.field_description and sonata_admin.field_description.hasAssociationAdmin %}
+            // Automatically select the created record after closing the popup window
+            {% if sonata_admin.field_description 
+            and sonata_admin.field_description.hasAssociationAdmin 
+            and btn_add 
+            and sonata_admin.field_description.associationadmin.hasRoute('create')
+            and sonata_admin.field_description.associationadmin.hasAccess('create') %}
```

Comment:
Can you please indent all the "and"s? I didn't notice it within the textarea, sorry…

Reply:
Done